src/rules/__tests__/prefer-importing-jest-globals.test.ts

```diff
@@ -0,0 +1,225 @@
+import { TSESLint } from '@typescript-eslint/utils';
+import dedent from 'dedent';
+import rule from '../prefer-importing-jest-globals';
+import { espreeParser } from './test-utils';
+
+const ruleTester = new TSESLint.RuleTester({
+  parser: espreeParser,
+  parserOptions: {
+    ecmaVersion: 2015,
+  },
+});
+
+ruleTester.run('prefer-importing-jest-globals', rule, {
```

Comment:
Please include the `line` in all your invalid cases, and you're missing more tests:
  * what if we're importing from default
  * what if we're using a name from not `@jest/globals` (i.e. `const describe = require('./mine')`)
  * what if we have our require at the end of the file (i.e. `const it = require('@jest/globals')` - while silly, that is valid if everything is wrapped in a `describe`)
  * what if template literals are used, and what if those have interopation?

A number of those will come out from my comments down below :)

(also please chuck in some more `import` based tests to mirror the `require` ones, since those involve a very different AST)

Reply:
I did reflect more tests for `import` and added the `line` for the invalid cases.

The default import is not working as expected, see the failing test case:

```js
{
  code: dedent`
    const describe = require('@jest/globals');
    describe("suite", () => {
      test("foo");
      expect(true).toBeDefined();
    })
  `,
  output: dedent`
    const { describe, test, expect } = require('@jest/globals');
    describe("suite", () => {
      test("foo");
      expect(true).toBeDefined();
    })
`,
  errors: [
    {
      endColumn: 7,
      column: 3,
      line: 3,
      messageId: 'preferImportingJestGlobal',
    } as const,
  ],
},
```

<hr>

```
Received:
      "const { test, expect } = require('@jest/globals');
```

What would be your recommendation to solve this? 

I thought of trying to use `VariableDeclarator`, `Identifier` but it's hard to grasp.